projects/kubernetes-sigs/etcdadm/patches/0014-Add-retry-on-listing-members-in-join-command.patch

```diff
@@ -0,0 +1,90 @@
+From 35a9d8d692c24b89be947ba216980676670668fa Mon Sep 17 00:00:00 2001
+From: Wonkun Kim <wonkun@amazon.com>
+Date: Fri, 3 Jun 2022 15:33:17 -0500
+Subject: [PATCH] Update join.go
+
+---
+ cmd/join.go | 51 +++++++++++++++++++++++++++++++--------------------
+ 1 file changed, 31 insertions(+), 20 deletions(-)
+
+diff --git a/cmd/join.go b/cmd/join.go
+index dee7eba9..fa503318 100644
+--- a/cmd/join.go
++++ b/cmd/join.go
+@@ -132,6 +132,22 @@ func membership() phase {
+ 				return nil
+ 			}
+ 
++			var lastErr error
++			retrySteps := 1
++			if in.etcdAdmConfig.Retry {
++				retrySteps = constants.DefaultBackOffSteps
```

Comment:
Why does this have to be in a condition? Do we want the retrySteps to be 1 if this is not configured?

Reply:
DefaultBackOffSteps is 5 and we want retrySteps to be one if Retry is false not to retry the step.